### PR TITLE
OEL-600: Move from runner module dependencies.

### PR DESCRIPTION
**Pull request by escuriola**
- https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-600

### CODE REVIEW

File: oe_showcase.info.yml [hunk 1]
```diff
@@ -2,14 +2,14 @@ name: OpenEuropa Showcase Profile
 type: profile
 description: 'Install with commonly used features for OpenEuropa Showcase profile.'
 core_version_requirement: ^8.9 || ^9.1
-install:
+dependencies:
   - block
   - block_content
```
Comment (abel-santos-corral) [inline oe_showcase.info.yml]: Are we using the block content to create custom blocks? Is this necessary here or can be removed?

File: runner.yml.dist [hunk 2]
```diff
@@ -15,18 +15,9 @@ drupal:
...
-    - "./vendor/bin/drush en easy_breadcrumb -y"
-    - "./vendor/bin/drush en oe_bootstrap_theme_helper -y"
-    - "./vendor/bin/drush en oe_whitelabel_helper -y"
     - "./vendor/bin/drush config-set system.theme default oe_whitelabel -y"
     - "./vendor/bin/drush config-set system.theme admin seven -y"
     - "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y"
-    - "./vendor/bin/drush en oe_multilingual -y"
     - "./vendor/bin/drush oe-multilingual:import-local-translations"
     # Don't aggregate or some pages will not be showed properly.
     - "./vendor/bin/drush -y config-set system.performance css.preprocess 0"
```
Comment (abel-santos-corral) [inline runner.yml.dist:23]: Please, add a folder (or check if after OEL-542 is already present, do a git rebase) of congig and export the configs of the system.performance and leave this things configured.

**Review by abel-santos-corral — CHANGES_REQUESTED** [review]

File: runner.yml.dist (hunk 2 above)
Comment (drishu) [reply runner.yml.dist:23]: these are definitely dev settings and should stay in runner

**Review by drishu — APPROVED** [review]

**Review by abel-santos-corral — APPROVED** [review]
Approved

**Pull request merged**